gyronimo/metrics/metric_covariant.hh

```diff
@@ -20,6 +20,7 @@
 #ifndef GYRONIMO_METRIC_COVARIANT
 #define GYRONIMO_METRIC_COVARIANT
 
+#include <gyronimo/core/contraction.hh>
```

Comment:
Is this include necessary here?

Reply:
My mistake. Will fix in the next commit.